CMakeLists.txt

```diff
@@ -25,6 +25,10 @@ if(CUDA_FOUND)
     endif()
 endif()
 
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "^(arm.*|ARM.*)")
+    add_compile_options(-mfpu=neon-vfpv4)
```

Comment:
I have a doubt that Visual Studio supports such compiler flag. Maybe we set it not for VS only?

Reply:
> I have a doubt that Visual Studio supports such compiler flag. Maybe we set it not for VS only?

Unknown flags are ignored during compilation though. I checked the logs and everything looks OK to me.